### ISSUE TITLE
Force unstake all NFTs in Gemfarm

### ISSUE THREAD

**Issue by tonyzheng11** [gemworks/gem-farm#135]
we are thinking about converting our project to pnfts and are considering if it was possible to force unstake all NFTs in the gemfarm before we converted. Is this possible as the bank manager?

**Comment by Exxempt** [comment]
No, the transfer has to be initiated by the owner of the vault. You can update to pNFTs and have it work.

**Comment by CCC1015** [comment]
I need to do this i have authority wallet, any advice?

**Comment by tschifra** [comment]
@Exxempt How to?
